### PR TITLE
Phish List

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by Zachinquarantine** [comment]
can someone check this pr please? @spirillen @mitchellkrogza

File: add-link [hunk 1]
```diff
@@ -38,3 +38,4 @@ https://www.mdscbenin.org/wp-includes/theme-compat/dar/newsant1/login.php
 https://nort2on.com/setup/
 http://eternalstarculture.com/intern/home.php
 https://tinyurl.com/4cbf4wcp
+https://www.kunestra.hr/.uk/
```
Comment (spirillen) [inline add-link:41]: Confirmed, but you should be aware it is also working on all four versions of 

```js
http(s)?://(www.)?kunestra.hr/.uk/
```

image

File: add-domain [hunk 2]
```diff
@@ -522,3 +522,15 @@ pdfkinghub.xyz
 tracebb.trackingsystem.xyz
 postgo.webpostgo.icu
 totaltrkonups2298.fitritomoastramencostea.com
+www.roblox.com.es
```
Comment (spirillen) [inline add-domain:525]: Can't confirm

image

File: add-domain [hunk 3]
```diff
@@ -522,3 +522,15 @@ pdfkinghub.xyz
...
+newappmail.discoverynevers.com
```
Comment (spirillen) [inline add-domain:526]: Seems to have been solved

image

File: add-domain [hunk 4]
```diff
@@ -522,3 +522,15 @@ pdfkinghub.xyz
...
+app-uniswap.org-vote-eth-swap.site
```
Comment (spirillen) [inline add-domain:527]: Seems dead to me from both manual lookup and @pyfunceble

image

File: add-domain [hunk 5]
```diff
@@ -522,3 +522,15 @@ pdfkinghub.xyz
...
+app-uniswap.org-claim-eth-liquidty.site
```
Comment (spirillen) [inline add-domain:528]: Seems dead to me from both manual lookup and @pyfunceble

image

**Review by spirillen — CHANGES_REQUESTED** [review]
I have now spend 45 trying to verify this bulk load of domains, and the main part of those I've tested seems dead.

Please take a second look at your submission

File: add-domain [hunk 6]
```diff
@@ -522,3 +522,11 @@ pdfkinghub.xyz
 tracebb.trackingsystem.xyz
 postgo.webpostgo.icu
 totaltrkonups2298.fitritomoastramencostea.com
+tracktt.deliveryoffer.top
```
Comment (spirillen) [inline add-domain:525]: drill -T deliveryoffer.top        
.       518400  IN      NS      a.root-servers.net.
.       518400  IN      NS      j.root-servers.net.
.       518400  IN      NS      k.root-servers.net.
.       518400  IN      NS      c.root-servers.net.
.       518400  IN      NS      m.root-servers.net.
.       518400  IN      NS      e.root-servers.net.
.       518400  IN      NS      i.root-servers.net.
.       518400  IN      NS      l.root-servers.net.
.       518400  IN      NS      h.root-servers.net.
.       518400  IN      NS      b.root-servers.net.
.       518400  IN      NS      g.root-servers.net.
.       518400  IN      NS      d.root-servers.net.
.       518400  IN      NS      f.root-servers.net.
top.    172800  IN      NS      a.zdnscloud.com.
top.    172800  IN      NS      b.zdnscloud.com.
top.    172800  IN      NS      c.zdnscloud.com.
top.    172800  IN      NS      d.zdnscloud.com.
top.    172800  IN      NS      f.zdnscloud.com.
top.    172800  IN      NS      g.zdnscloud.com.
top.    172800  IN      NS      i.zdnscloud.com.
top.    172800  IN      NS      j.zdnscloud.com.
top.    3600    IN      SOA     a.zdnscloud.com. td_dns_gtld.knet.cn. 1626793776 600 200 2491200 3600

File: add-domain [hunk 7]
```diff
@@ -522,3 +522,11 @@ pdfkinghub.xyz
...
+mail-e9eredelivery.com
```
Comment (spirillen) [inline add-domain:526]: drill -T mail-e9eredelivery.com
.       518400  IN      NS      m.root-servers.net.
.       518400  IN      NS      f.root-servers.net.
.       518400  IN      NS      e.root-servers.net.
.       518400  IN      NS      b.root-servers.net.
.       518400  IN      NS      k.root-servers.net.
.       518400  IN      NS      c.root-servers.net.
.       518400  IN      NS      g.root-servers.net.
.       518400  IN      NS      h.root-servers.net.
.       518400  IN      NS      d.root-servers.net.
.       518400  IN      NS      j.root-servers.net.
.       518400  IN      NS      i.root-servers.net.
.       518400  IN      NS      a.root-servers.net.
.       518400  IN      NS      l.root-servers.net.
com.    172800  IN      NS      k.gtld-servers.net.
com.    172800  IN      NS      g.gtld-servers.net.
com.    172800  IN      NS      e.gtld-servers.net.
com.    172800  IN      NS      h.gtld-servers.net.
com.    172800  IN      NS      l.gtld-servers.net.
com.    172800  IN      NS      i.gtld-servers.net.
com.    172800  IN      NS      d.gtld-servers.net.
com.    172800  IN      NS      a.gtld-servers.net.
com.    172800  IN      NS      m.gtld-servers.net.
com.    172800  IN      NS      j.gtld-servers.net.
com.    172800  IN      NS      c.gtld-servers.net.
com.    172800  IN      NS      b.gtld-servers.net.
com.    172800  IN      NS      f.gtld-servers.net.
com.    900     IN      SOA     a.gtld-servers.net. nstld.verisign-grs.com. 1626793872 1800 900 604800 86400

File: add-domain [hunk 8]
```diff
@@ -522,3 +522,11 @@ pdfkinghub.xyz
...
+trackpp.searchmyitem.xyz
```
Comment (spirillen) [inline add-domain:527]: drill -T searchmyitem.xyz
.       518400  IN      NS      a.root-servers.net.
.       518400  IN      NS      b.root-servers.net.
.       518400  IN      NS      c.root-servers.net.
.       518400  IN      NS      d.root-servers.net.
.       518400  IN      NS      e.root-servers.net.
.       518400  IN      NS      f.root-servers.net.
.       518400  IN      NS      g.root-servers.net.
.       518400  IN      NS      h.root-servers.net.
.       518400  IN      NS      i.root-servers.net.
.       518400  IN      NS      j.root-servers.net.
.       518400  IN      NS      k.root-servers.net.
.       518400  IN      NS      l.root-servers.net.
.       518400  IN      NS      m.root-servers.net.
xyz.    172800  IN      NS      x.nic.xyz.
xyz.    172800  IN      NS      generationxyz.nic.xyz.
xyz.    172800  IN      NS      z.nic.xyz.
xyz.    172800  IN      NS      y.nic.xyz.
xyz.    3600    IN      SOA     ns0.centralnic.net. hostmaster.centralnic.net. 3000620592 900 1800 6048000 3600

File: add-domain [hunk 9]
```diff
@@ -522,3 +522,11 @@ pdfkinghub.xyz
...
+steamcomnnulty.com
```
Comment (spirillen) [inline add-domain:528]: Looks alive :+1: 

image

File: add-domain [hunk 10]
```diff
@@ -522,3 +522,11 @@ pdfkinghub.xyz
...
+yahoofbuyv.weebly.com
```
Comment (spirillen) [inline add-domain:529]: Taken down, See also Special Rules in Pyfunceble 

https://github.com/funilrys/PyFunceble/blob/89f2b69c4f02657124d3a05d6b73e43ca5a46c06/PyFunceble/checker/availability/extra_rules.py#L99-L101

from commit https://github.com/funilrys/PyFunceble/commit/89f2b69c4f02657124d3a05d6b73e43ca5a46c06

File: add-domain [hunk 11]
```diff
@@ -522,3 +522,11 @@ pdfkinghub.xyz
...
+attonlineservicexx.weebly.com
+julyemailclosurer.weebly.com
```
Comment (spirillen) [inline add-domain:531]: `pyfunceble -d julyemailclosurer.weebly.com attonlineservicexx.weebly.com yahoofbuyv.weebly.com`

You are using the Beta version of PyFunceble 4.0.0!
Please take the time to communicate with us when you notice
something unusual.




Subject                                                                                              Status      Source    
---------------------------------------------------------------------------------------------------- ----------- ----------
`attonlineservicexx.weebly.com`                                                                        INACTIVE    SPECIAL   
`julyemailclosurer.weebly.com`                                                                         INACTIVE    SPECIAL   
`yahoofbuyv.weebly.com `                                                                               INACTIVE    SPECIAL

File: add-domain [hunk 12]
```diff
@@ -522,3 +522,11 @@ pdfkinghub.xyz
...
+support-1oakyf-league-of-legends.000webhostapp.com
```
Comment (spirillen) [inline add-domain:532]: Active:

image


You can test this with the special rules for @PyFunceble: 

https://github.com/funilrys/PyFunceble/blob/89f2b69c4f02657124d3a05d6b73e43ca5a46c06/PyFunceble/checker/availability/extra_rules.py#L84-L86

**Review by spirillen — CHANGES_REQUESTED** [review]
I asked you to take a second look at these domains, there are still a lot of :skull: once.

This is exactly one of many reasons I do one domain = one issue at https://mypdns.org/my-privacy-dns to make sure you can get stuff merged and distributed A.S.A.P

**Comment by spirillen** [comment]
@Zachinquarantine I can recommend you to use @PyFunceble to test submissions before adding them. PyFunceble is written for Linux, but it have with some success been used on windows within PowerShell and Cygwin by @DandelionSprout

PyFunceble at GH: https://github.com/funilrys/PyFunceble
PyFunceble at MypDNS (mirror): https://mypdns.org/pyfunceble/PyFunceble
PyFunceble at pypi: https://pypi.org/project/PyFunceble-dev/
PyFunceble at Read the docs: https://pyfunceble.readthedocs.io/en/dev/

**Comment by Zachinquarantine** [comment]
@spirillen

File: add-domain [hunk 13]
```diff
@@ -522,3 +522,10 @@ pdfkinghub.xyz
 tracebb.trackingsystem.xyz
 postgo.webpostgo.icu
 totaltrkonups2298.fitritomoastramencostea.com
+tracktt.deliveryoffer.top
```
Comment (spirillen) [inline add-domain:525]: Note for others.

This domain seems inactive at the writing moment; However the domain is still active, which means the DNS/web-server etc can be activated within milliseconds or only be response to specific regions / IP addresses, Hence leaving this record as ACTIVE

File: add-domain [hunk 14]
```diff
@@ -522,3 +522,10 @@ pdfkinghub.xyz
...
+mail-e9eredelivery.com
```
Comment (spirillen) [inline add-domain:526]: Note for others.

This domain seems inactive at the writing moment; However the domain is still active, which means the DNS/web-server etc can be activated within milliseconds or only be response to specific regions / IP addresses, Hence leaving this record as ACTIVE

**Review by spirillen — APPROVED** [review]
Approved with comments for others to read

**Pull request merged**